### PR TITLE
Fix coverage of specs on CI

### PR DESCRIPTION
**Pull request by skeleton**
Since we updated "phpspec/phpspec" to 3.0, build on our CI to test corevage does not work.

- [x] Update phpunit to `5.4.*` (not the last version because of one of these requirements is php >= 7.0) 
- [x] Fix phpunit tests
- [x] Add all suites test to calculate the coverage

### CODE REVIEW

**Comment by ClementGautier** [comment]
GTM

File: app/phpspec.jenkins-coverage.yml [hunk 1]
```diff
@@ -1,7 +1,192 @@
 formatter.name: dot
```
Comment (nidup) [inline app/phpspec.jenkins-coverage.yml]: we can't use phpspec.yml.dist to avoid to duplicate the conf

File: composer.json [hunk 2]
```diff
@@ -61,7 +61,7 @@
         "box/spout": "2.5.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "3.7.*",
+        "phpunit/phpunit": "5.4.*",
```
Comment (nidup) [inline composer.json]: :+1:

File: composer.json [hunk 3]
```diff
@@ -79,7 +79,8 @@
         "sensiolabs/behat-page-object-extension": "1.0.1",
         "phpspec/phpspec": "3.0.0",
         "akeneo/phpspec-skip-example-extension": "2.0.0",
-        "akeneo/php-coupling-detector": "dev-master"
+        "akeneo/php-coupling-detector": "dev-master",
+        "henrikbjorn/phpspec-code-coverage": "3.0.*"
```
Comment (nidup) [inline composer.json]: ok like for https://github.com/akeneo/ExcelInitBundle/blob/master/composer.json#L29 for instance

**Comment by nidup** [comment]
question about phpspec conf then GTM

**Pull request merged**